packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1405,6 +1416,19 @@ export default class TradeStore extends BaseStore {
     get is_vanilla() {
         return this.contract_type === 'vanilla';
     }
+    async getActionToastbox(response) {
+        const list = getAvailableContractTypes(this.contract_types_list, unsupported_contract_types_list);
+        return (this.action_toastbox = {
+            key: true,
+            buy_price: formatMoney(this.root_store.client.currency, response.buy_price, true, 0, 0),
+            contract_type: this.contract_type,
+            currency: getCurrencyDisplayCode(this.root_store.client.currency),
+            list,
+        });
+    }
+    async actionChangeToastbox(is_visible) {
```

Comment:
`actionChangeToastbox` doesn't sound good. Can we consider `clearContractPurchaseToastBox`?

Reply:
why are we passing an empty object(`{}`) to `is_visible`.

Reply:
resolved